mmv1/third_party/terraform/utils/common_diff_suppress_test.go.erb

```diff
@@ -285,3 +285,37 @@ func TestDurationDiffSuppress(t *testing.T) {
 		}
 	}
 }
+
+func TestLastSlashDiffSuppress(t *testing.T) {
+	cases := map[string]struct {
+		Old, New           string
+		ExpectDiffSuppress bool
+	}{
+		"locations to zones": {
```

Comment:
Do we want to change the names of these tests? I'm assuming they were copied and pasted :)

Reply:
resolved